utils/api/benefitHandler.ts

```diff
@@ -882,6 +882,15 @@ export class BenefitHandler {
               allResults.partner.gis.cardDetail.collapsedText.push(
                 this.translations.detailWithHeading.partnerEligible
               )
+              if (
+                this.input.partner.invSeparated &&
```

Comment:
not needed the whole block is for invSeparated = true

Reply:
fixed